tests/models/test_mtp.py

```diff
@@ -0,0 +1,138 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+from pymatgen.core import Structure
+
+from crystal_diffusion.models.mtp import MTPWithMLIP3
+
+
+class MockStructure:
```

Comment:
This is more of a style nitpick, so feel free to ignore. 
I would tend to reserve the word "Mock" for things that come from the Mock library (or the mocker fixture). Here, I would call this "FakeStructure", or "StructureStub"...

Reply:
changed, that's a good idea